File: Algorithms/Easy/1304_FindNUniqueIntegersSumUpToZero/Solution.rb
# @param {Integer} n
# @return {Integer[]}
def sum_zero(n)
    answer = []
    answer << 0 if (n % 2 == 1)
    n /= 2
    n.times { |i| answer << (i + 1) << -(i + 1) }
    answer
end

File: Algorithms/Easy/1304_FindNUniqueIntegersSumUpToZero/Solution.py
class Solution(object):
    def sumZero(self, n):
        """
        :type n: int
        :rtype: List[int]
        """
        answer = [i for i in range(1, (n // 2) + 1)]
        answer += [-i for i in answer]

        return answer if (n % 2 == 0) else answer + [0]